presto-main/src/test/java/io/prestosql/operator/scalar/AbstractTestRegexpFunctions.java

```diff
@@ -221,6 +238,13 @@ public void testRegexpExtractAll()
     @Test
     public void testRegexpSplit()
     {
+        assertFunction(
+                "REGEXP_SPLIT('abc有朋$%X自9远方来💰', '')",
+                new ArrayType(createVarcharType(14)),
+                ImmutableList.of("", "a", "b", "c", "有", "朋", "$", "%", "X", "自", "9", "远", "方", "来", "💰", ""));
```

Comment:
i guess this is pre-existing, but i don't underdstand why would split want to return `""` on both ends.

Reply:
I guess you match beginning of string, then on the left you have empty string and on the right you have a letter.

Similarly, `"REGEXP_SPLIT('abc', 'a')` would give you: `"", "bc`